src/Controllers/CrudTraits/CrudIndexTrait.php

```diff
@@ -43,13 +54,24 @@ public function index(Request $request)
             }
         }
 
+        // FILTER NULL OR NOT NULL -------------
+        if (!empty($request->get('is_empty'))) {
```

Comment:
can you put example here what do we expect the request to look like?
we expect `is_empty=something&is_empty.column_name?`
we had an idea were it's csv columns like `is_empty=column1,column2` in request but I'm not sure about the usage of `"is_empty.{$column}"` we have in line 63 below this

Reply:
from slack 
```
is_empty[column_name]=true&is_empty[another_col]=false
```